server/models/group.js

```diff
@@ -20,4 +27,24 @@ const GroupSchema = new mongoose.Schema({
   }
 });
 
+GroupSchema.statics = {
+  /**
+   * Get group
+   * @param {GroupName} - the name of the group
+   * @return {Promise<Group, APIError>}
+   */
+
+  get(name) {
+    return this.findOne({ name: name })
+      .exec()
+      .then((group) => {
+        if (group) {
+          return group;
```

Comment:
I see that the `.exec()` will return a promise, but how does this line work once you return it? Will later `.then(...)` blocks be able to be chained together?

ie. 
``` javascript
Group.get('name')
    .then(name => console.log(name));
```